src/types.ts

```diff
@@ -342,6 +342,8 @@ export interface GetSwapQuoteRequestParams {
     gasPrice?: BigNumber;
     excludedSources?: ERC20BridgeSource[];
     affiliateAddress?: string;
+    apiKey?: string;
```

Comment:
Do we need this added to _both_ `GetSwapQuoteRequestParams` _and_ `CalculateSwapQuoteParams`?

Reply:
Consider pushing these options under an `rfqt:` namespace.

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/902b15d5dc6ffc311162e25c210299ed5b38535b